### DISCUSSION TITLE
The Network page is too slow

### DISCUSSION THREAD

**Discussion by jaymeyer** [community/community#53419] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I use the Insights > Network page literally every day on my team.  We have active committers, I need to see where my changes are vs. other people's branches, etc.  This view is so useful and important.  
https://github.com/theOrg/therepo/network

But it takes 10-30 seconds to load.

I've used other tools like Bitbucket which have no speed problems, and they include things like the committer, a readable timestamp, and a vertical view of the timeline that I happen to prefer.  But even though it has more information on this single screen, it's instantaneous - which is nice.  If github's Network screen drew the graph faster, that would make me much happier to look at my code graph each day.

**Comment by tyirvine** [comment]
I'm bumping this. It could use an update. Maybe we could even add a separate `Activity` tab? Something like the extension [Git Graph](https://github.com/mhutchie/vscode-git-graph) for vscode would be much better. Just having more space and a vertical scroll rather than a horizontal scroll would be great. Here's what it looks like. ⤵︎

<img width="503" alt="Screenshot 2023-05-07 at 3 27 10 PM" src="https://user-images.githubusercontent.com/39813066/236703416-da26bc8b-9b6f-4576-8253-044c4eca1845.png">

**Reply by jaymeyer** [reply]
this would be very nice: more information on one screen: timestamp, committer, color-coded branch graph.
The only thing this view is missing is the branch names: e.g. main vs. feature-new-thing.
See bitbucket for a nice vertical-oriented view of the commit graph.

**Reply by tyirvine** [reply]
Ah I see, yeah! The Bitbucket graph looks great.

**Comment by jaymeyer** [comment]
apparently I don't understand : How do I tell the developers of Github that I want them to make a change to the github software?  Is this the wrong place?

Should I somehow open some sort of "issue" or "bug report"? How does that work, or is it typical to have these go "unanswered"?

**Reply by tyirvine** [reply]
From the README.md. ⤵︎

> ### From a suggestion to a shipped feature
> Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission. From there, we will work with you, and the entire community, to ensure we understand the current capabilities GitHub doesn’t have and explore the space for potential solutions to your problem statement:
> 
> If the product team determines that we are going to prioritize a feature to solve the problem you've identified, we may open an issue and track its development in the [public roadmap](https://github.com/github/roadmap).
If the product team determines that we will not be working to solve the problem you have identified, we may comment on the discussion describing our reasoning so our decisions can remain transparent.

So it seems you'll have to wait for more interest in this issue. Or try to garner interest in it to catch the attention of the GitHub team. If you find similar issues to this it'd probably be wise to link them to this issue and vice versa.

**Comment by jaymeyer** [comment]
and today the page literally never draws, just spins for 10 minutes until I refresh, complete fail.
